converged_sound_open_fw_modules_development/FW/intel_common/module_binmaps/downstream_module.binmap

```diff
@@ -0,0 +1,25 @@
+module b DOWNSTR
```

Comment:
What is this file used for ? It looks like we can use this to generate the extended manifest so this these should probably go under tools/manifest.

Reply:
Btw, is there any docs on the binmap syntax ?

Reply:
@lgirdwood 
I slimmed down this patch a little bit, but still have to think more how to split it into smaller chunks. 

Binmap files are by adsp_fw_builder to create manifest. At the moment the only docs on binmap syntax is placed in README file.

Reply:
ok, so binmap is like the toml files we have in rimage today ? Is it based on any Internet format or does it have any parsing rules we should have everyone follow ?

Reply:
To be honest I do not know what we have exactly in toml files, but binmaps are configuration files for modules - it is rather the internal format - at the moment we have described it only in README.md

Reply:
It looks like they are the same thing as toml for rimage, they are quite similar (except toml uses = and "" for setting values). We need a plan to align these.

Reply:
> It looks like they are the same thing as toml for rimage, they are quite similar (except toml uses = and "" for setting values). We need a plan to align these.

The plan for loadable modules cross compatibility is to use adsp_fw_builder tool instead of rimage so the description alignment in that area should not be critical.